lit-config.js

```diff
@@ -1,21 +1,11 @@
-const { sortMemberRules } = require('./sort-member-config');
-
 module.exports = {
-	"extends": "./index.js",
-	"parser": "babel-eslint",
+	"extends": "./web-config.js",
 	"env": {
-		"browser": true,
 		"es6": true
 	},
 	"plugins": [
 		"lit",
```

Comment:
Not sure if the extra trailing comma here will be an issue -- I _think_ it's now allowed everywhere.